### PR TITLE
Audio structure

### PR DESCRIPTION
**Pull request by FelSiq**
Thats a (big) improvement over #5. It has the same logic behind, but has more technical details implemented. Don't take nothing implement here as a final decision. It is just a advanced prototype and should be modified accordingly to the discussion #6 and possibly #7.

### CODE REVIEW

File: solomon/bytestream.h [hunk 1]
```diff
@@ -6,21 +6,48 @@
...
 // Typedef and define section
+#define TEMP_SERVER_DIR "./.SERVER_DIR/"
+
+// Note that just this path isn't suffice for
+// the client, as it's possible to exists n clients
+// for a single server.
+#define TEMP_CLIENT_DIR "./.CLIENT_DIR/"
+
+// Calculated as 
+//	  ceil(log10(1 + MAX_UNSIGNED_LONG_LONG_INT))
```
Comment (ET33) [inline solomon/bytestream.h:17]: I had trouble trying to understand why use this formula, and I concluded it's the max number of digits a `unsigned long long int` would assume, right? If so, make a comment explaining this.

**Review by ET33 — APPROVED** [review]
I've approved, but there's a comment out there. Take a look please.

File: solomon/bytestream.c [hunk 2]
```diff
@@ -165,15 +376,29 @@ sound_struct *processSounds(data_unit *cur_data_unit, int *process_end) {
...
+
+	if (play_audio) {
+		// Should sounds be played as long as they received?
+		// This flag is supposed to be 0 (false) for the Server 
+		// and any other value (true) for every Client.
+		pthread_create(ss->thread_id + PLAY_MICROAUDIOS, NULL, play_microaudios, (void *) &(ss->args));
+	}
+
 
 	// Join (wait) threads terminate
```
Comment (ET33) [inline solomon/bytestream.c]: Confusing comment

**Review by ET33 — CHANGES_REQUESTED** [review]

**Review by ET33 — APPROVED** [review]
:shipit:

File: solomon/bytestream.h (hunk 1 above)
Comment (FelSiq) [reply solomon/bytestream.h:17]: > concluded it's the max number of digits a unsigned long long int would assume, right?

Yes, that's exactly what it is supposed to be. My fault not explaining it better.

**Pull request merged**